src/sentry/static/sentry/app/utils/withCommitters.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
+
+import {Client} from 'app/api';
+import {Organization, Project, Event, Group, Committer} from 'app/types';
+import {getCommitters} from 'app/actionCreators/committers';
+import CommitterStore from 'app/stores/committerStore';
+import getDisplayName from 'app/utils/getDisplayName';
+
+type DependentProps = {
+  api: Client;
+
+  organization: Organization;
+  project: Project;
+  event: Event;
+  group?: Group;
+};
+
+// XXX: State does not include loading/error because components using this
+// HOC (suggestedOwners, eventCause) do not have loading/error states. However,
+// the store maintains those states if it is needed in the future.
+type InjectedProps = {
+  committers: Committer[];
+};
+
+const INITIAL_STATE: InjectedProps = {
```

Comment:
Can we treat acronyms/abbreviations as words to be consistent? So this should be initialState

Reply:
Oh yea, you're right 😄